README.md

```diff
@@ -9,7 +9,6 @@ The resulting library is a "fat" library suitable for multiple architectures. Th
 - armv7s (iOS)
 - arm64 (iOS)
 - i386 (iOS Simulator)
-- x86_64 (iOS Simulator)
```

Comment:
If we want, we could rewrite this because our library is purposely NOT fat (since fat would not be shippable to app store)

also: ahead of local plans release, should we try to submit a patch build at some point that just contains this library added and make sure it passes their review? or is that not a big concern?